dist/profile/templates/buildmaster/casc/clouds.yaml.erb

```diff
@@ -127,9 +127,11 @@ jenkins:
         remoteAdmin: "<%= @agents_setup[agent["os"].to_s]["remoteAdmin"] %>"
         remoteFS: "<%= @agents_setup[agent["os"].to_s]["agentDir"] %>"
         securityGroups: "ci-agents"
+        <%- if agent["spot"] -%>
```

Comment:
Note: on azure VMs, the test is done on `agent["spotInstance"] || agent["spot"]`, I've kept the later only.

Reply:
Keeping `spot` make sense 👍